common/include/pcl/common/impl/accumulators.hpp

```diff
@@ -35,8 +35,7 @@
  *
  */
 
-#ifndef PCL_COMMON_IMPL_DETAIL_ACCUMULATORS_HPP
-#define PCL_COMMON_IMPL_DETAIL_ACCUMULATORS_HPP
+#pragma once
```

Comment:
Please be consistent in either leaving an empty line or not between the top comment and pragma decl

Reply:
Okay, I had simply run a script so the places that already had a space originally still have them. Will change it to leave a space always.

Reply:
Got it. If you share the script, it'll be easier to review your PR

Reply:
I have updated the description of this PR with the link.